### ISSUE TITLE
Autocompletion for

### ISSUE THREAD

**Issue by zoechi** [dart-lang/angular_analyzer_plugin#141]
Just ideas because I saw other autocompletion issues being added

- `[style.|`, `style.|`
- `[style.xxx.|`, `style.xxx.|`

Not sure if anything useful can be provided for
- `[class.|`, `class.|`
- `[attr.|`, `attr.|`

**Comment by MichaelRFairhurst** [comment]
Definitely! we could autocomplete property names and units, since nobody has a css reference book in their head would definitely be useful

**Comment by zoechi** [comment]
Just thinking. I don't know if `style.xxx="someString"` or `style.xxx="{{someString}}"` is supported.
